src/Containers/DropArea/DropArea.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+import { DropArea, IDropAreaProps } from './DropArea';
+
+export default {
+    title: 'Components/DropArea',
+    component: DropArea,
+    args: {},
+    argTypes: {
+        onDragLeave: { action: 'leaving drop are' },
+        onDragEnter: { action: 'entering drop area' },
+        onDropHandler: { action: 'files dropped' },
+        onClickHandler: { action: 'you clicked!' },
+    },
+} as Meta;
+
+export const Basic: Story<IDropAreaProps> = (args) => <DropArea {...args} />;
+export const BigWidth = Basic.bind({});
+BigWidth.args = {
+    ...Basic.args,
+    width: 500,
+};
+export const IsDisabledTrue = Basic.bind({});
+IsDisabledTrue.args = {
+    ...Basic.args,
+    isDisabled: true,
+};
+export const TestPreventDefault = Basic.bind({});
```

Comment:
Remove this... the disabled state tests this

Reply:
no. this is not true.

Reply:
What is the user test you are trying to do here?

Reply:
when you click the Browse Files button, the default behaviour is to open dialog box to choose file. with the hacky CSS I can, and anyone using this component can, prevent this default behavior to happen (that is, not open dialog box). without the hacky CSS I can't, and always will open this dialog box, apart from doing whatever other thing you program on the onClick event handler (everything except preventing this default behavior). the disabled state removes any interaction with the component. so essentially suppose a developer wants to do something else than opening the dialog box when the user clicks, that is, show a message instead or whatever other thing but not open the dialog box to choose file. with the actual implementation he can't, but with the hacky CSS he can. disabled state doesn't allow this because remove any kind of interaction (all interactions).

Reply:
so essentially the test or story I am trying to implement is TestDoNotOpenTheDialgoBoxToChooseFileButDoThisOtherThingInsteadWhenClicking